src/app/shared/components/footer/footer.component.ts

```diff
@@ -37,4 +45,27 @@ export class FooterComponent implements OnInit {
     window.scrollTo(0, 0);
   }
 
+  getCurrentUser() {
+    this.userService.userLoaded$.subscribe((data) => {
+      if (data) {
+        this.currentUser = this.userService.currentUser;
+      } else {
+        this.currentUser = null;
+      }
+    });
+  }
+
+  submitConsultation() {
+    if (!this.currentUser) {
+      this.router.navigateByUrl('/auth');
+      return;
+    } else {
```

Comment:
Readability: The `else` is not needed if the `if` condition is guaranteed to `return`  every time control flow reaches there

Reply:
Made the changes.